src/EventStore.Client/EventStoreClientBase.cs

```diff
@@ -85,10 +86,13 @@ public abstract class EventStoreClientBase :
 			return new(channel, caps, invoker);
 		}
 
-#pragma warning disable 1591
+
+		/// <summary>
+		/// Gets the current channel info in an inexpensive way.
+		/// </summary>
+		/// <param name="cancellationToken"></param>
```

Comment:
```suggestion
		/// Gets the current channel info
```

Reply:
Makes sense, removed them.

Reply:
i'd still prefer to remove 'in an inexpensive way' unless you particularly would like to keep it?